mender-client-acceptance-testing/Dockerfile

```diff
@@ -16,10 +16,26 @@ ENV LC_ALL=en_US.UTF-8 LANG=en_US.UTF-8 LANGUAGE=en_US.UTF-8
 COPY requirements-py3.txt .
 RUN pip3 install -r requirements-py3.txt
 
+# For warrior tests
+RUN apt-get install -y python-pip && \
+    apt-get install -qqy python-pip && \
```

Comment:
is this just duplicated or is it required?

Reply:
Duplicated :sweat_smile: